Makefile

```diff
@@ -14,3 +14,7 @@ Makefile: ;
 build/CMakeCache.txt:
 	@mkdir -p ${build}
 	@cmake -B ${build} -S . -G Ninja
+
+.PHONY: test-author-perl
+test-author-perl:
```

Comment:
I would rather add this target on CMake level so it also works with Ninja and can depend on the build if that's needed.

Reply:
but that didn't work like expected, like I wrote.
and it doesn't depend on the build, that's the point.
all the static checks don't depend on the build.

but if someone can explain the magic of the check target to me, and how we can make a target that doesn't run all the static checks, then I'm willing to go with cmake

Reply:
Add discussed short-term you should define an additional custom target in cmake/test-targets.cmake e.g. "test-author" and also exclude that in "check-pkg-build"

Reply:
Done